swagger_server/controllers/survey_response.py

```diff
@@ -127,6 +137,39 @@ def _get_collection_exercise(self, collection_exercise_id):
             ons_env.logger.debug('Collection Exercise not found')
         return collection_exercise
 
+
+    def _get_survey_service(self, survey_id):
```

Comment:
The function name is a bit misleading, we are getting the survey_data not the service